changelog/@unreleased/pr-343.v2.yml

```diff
@@ -0,0 +1,6 @@
+type: fix
+fix:
+  description: '`gradle-consistent-versions` now supports Gradle 6.0. The minimum
+    required Gradle version has been raised to 5.3'
```

Comment:
lies, please amend the required version in VersionsLockPlugin :)

Reply:
https://github.com/palantir/gradle-consistent-versions/pull/343/files#diff-2382390745b175d9f349bda21607d522R112